docs/image-sizes.md

```diff
@@ -0,0 +1,40 @@
+# Images Sizes
+
+You can define additional image thumbnail sizes with `add_image_size`.
+
+The custom thumbnail sizes should be declared in the callback function to the `after_setup_theme` action.
+
+Example:
+```php
+add_action( 'after_setup_theme', __NAMESPACE__ . '\\theme_setup' );
+function theme_setup() {
+    add_image_size( 'category-thumb', 300, 9999 ); // 300 pixels wide (and unlimited height)
+}
+```
+
+You can find more info at the [WordPress Developer Reference](https://developer.wordpress.org/reference/functions/add_image_size/) and [WordPress developer guide](https://developer.wordpress.org/themes/functionality/featured-images-post-thumbnails/#add-custom-featured-image-sizes)
```

Comment:
Once the `add_image_size()` function is documented here let's ditch the link to the WP developer reference. I'm a bit torn on whether to include the WP developer guide link but I guess it has a bunch of extra detail.